detection-rules/link_google_translate.yml

```
@@ -10,7 +10,10 @@ authors:
 severity: "low"
 source: |
   type.inbound
-  and any(body.links, .href_url.domain.root_domain == "translate.goog")
+  and any(body.links,
+      .href_url.domain.root_domain == "translate.goog"
+      or "google_translate_open_redirect" in .href_url.rewrite.encoders
+  )
```

sweet! well in that case...
```suggestion
  and any(body.links, "google_translate_open_redirect" in .href_url.rewrite.encoders)
```